packages/fxa-content-server/server/lib/beta-settings.js

```diff
@@ -0,0 +1,113 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { readFileSync } = require('fs');
+const { join } = require('path');
+const { createProxyMiddleware } = require('http-proxy-middleware');
+const config = require('./configuration');
+
+let settingsIndexFile;
+const env = config.get('env');
+const settingsIndexPath = join(
+  __dirname,
+  '..',
+  '..',
+  config.get('static_directory'),
+  'beta',
+  'settings',
+  'index.html'
```

Comment:
I can't find this file.

Reply:
This is the location of the build file, so if you run `yarn build` it will production build everything and you'll find it at `dist/beta/settings/index.html`

Reply:
Thanks! TIL